integrations/osmosis/packages/proto-build/src/main.rs--

```diff
@@ -11,10 +11,13 @@ use proto_build::{
 };
 
 /// The Cosmos SDK commit or tag to be cloned and used to build the proto files
-const COSMOS_SDK_REV: &str = "origin/osmosis-main";
+const COSMOS_SDK_REV: &str = "osmosis-main";
 
 /// The osmosis commit or tag to be cloned and used to build the proto files
-const OSMOSIS_REV: &str = "origin/main";
+const OSMOSIS_REV: &str = "v17.0.0-rc0";
```

Comment:
Does anyone know why osmosis have this file?

Reply:
yes, it's how they generate Rust code from `.proto` files